src/varity/vcf_to_hgvs/protein.clj

```diff
@@ -430,8 +449,15 @@
                      format-alt-prot-seq
                      (subs (dec (+ ppos offset)))
                      (string/index-of "*"))]
-    (if (= alt \*)
+    (cond
+      (or ref-include-utr-ini-site-boundary
+          (not (string/starts-with? alt-prot-seq "M")))
```

Comment:
To exclude UTR and initiation site is changed but first amino acid of alt-prot-seq is "M".